omnibus/files/private-chef-ctl-commands/secrets.rb

```diff
@@ -27,13 +29,43 @@
   "opscode-reporting" => ["rabbitmq_password", "sql_password", "sql_ro_password"],
 }
 
+SERVICES_REQUIRING_RESTART = {
+  "bookshelf.access_key_id" => ["opscode-erchef", "bookshelf"],
+  "bookshelf.secret_access_key" => ["opscode-erchef", "bookshelf"],
+  "bookshelf.sql_password" => ["bookshelf"],
+  "chef-server.superuser_key" => ["opscode-reporting"],
+  "chef-server.webui_key" => ["oc_id"],
+  "chef-server.webui_pub_key" => ["opscode-erchef", "opscode-reporting"],
+  "data_collector.token" => ["opscode-erchef", "nginx"],
+  "ldap.bind_password" => ["opscode-erchef"],
+  "manage.secret_key_base" => ["chef-manage"],
+  "manage.secret_token" => ["chef-manage"],
+  "oc_bifrost.sql_password" => ["oc_bifrost"],
+  "oc_bifrost.superuser_id" => ["opscode-erchef", "oc_bifrost", "opscode-chef-mover"],
+  "oc_id.secret_key_base" => ["oc_id"],
+  "oc_id.sql_password" => ["oc_id"],
+  "opscode-reporting.rabbitmq_password" => ["opscode-reporting"],
+  "opscode-reporting.sql_password" => ["opscode-reporting"],
+  "opscode_erchef.sql_password" => ["opscode-erchef", "opscode-chef-mover"],
+  "push-jobs-server.pushy_priv_key" => ["opscode-push-jobs-server"],
+  "push-jobs-server.pushy_pub_key" => ["opscode-push-jobs-server"],
+  "push-jobs-server.sql_password" => ["opscode-push-jobs-server"],
+  "rabbitmq.actions_password" => ["opscode-erchef", "opscode-reporting"],
+  "rabbitmq.management_password" => ["opscode-erchef", "rabbitmq"],
```

Comment:
@srenatus We weren't sure if you meant we should remove this line altogether or if we should remove rabbitmq from the list of services to be restarted. We ended up doing the second one, so hopefully that's what you meant!